src/h5web/visualizations/heatmap/Tooltip.tsx

```diff
@@ -57,12 +55,9 @@ function Tooltip(props: Props): ReactElement {
   const onPointerOut = useCallback(hideTooltip, [hideTooltip]);
   const onPointerDown = useCallback(hideTooltip, [hideTooltip]);
 
-  // Trigger `pointermove` when user stops panning to show tooltip if pointer still intersects with mesh
-  const onPointerUp = useCallback(intersect, [intersect]);
```

Comment:
Small bug fix: when zoomed in, `intersect()` could return true even when outside the canvas so the popup would reappear when it shouldn't. Now, I just leave the tooltip hidden until the user starts to move the mouse again.